pytest-selenium/tests/test_attribution.py

```diff
@@ -49,3 +50,59 @@ def test_attribution_collapsed_by_default_expands_when_clicked(
     # AND: The attribution collapses on clicking it again
     attribution.click_attribution_link()
     assert not attribution.is_open
+
+
+@markers.test_case("C476304")
+@markers.parametrize(
+    "book_slug,page_slug", [("college-physics", "1-2-physical-quantities-and-units")]
+)
+@markers.nondestructive
+def test_attribution_collapses_on_navigating_to_new_page(selenium, base_url, book_slug, page_slug):
+
+    # GIVEN: A page URL in the format of {base_url}/books/{book_slug}/pages/{page_slug}
+    # AND: The citation/attribution tab is open
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    attribution = content.attribution
+    attribution.click_attribution_link()
+
+    # WHEN: Navigating via next link
+    content.click_next_link()
+
+    # THEN: The citation/attribution section is not open on the new page
+    assert not attribution.is_open
+
+    attribution.click_attribution_link()
+
+    # WHEN: Navigating via Back link
+    content.click_previous_link()
```

Comment:
i think it might be better to have 3 separate tests for next/prev/toc

Reply:
@TomWoodward 
Wouldn't that be too overwhelming to have just two lines of code in one test...I preferred this way because testrail test was designed like this to include all possible navigations in one test

Reply:
I removed the TOC part from this test since the TOC structure needs more time for me to put in a neat framework